src/liballoc_jemalloc/lib.rs

```diff
@@ -77,7 +77,8 @@ const MIN_ALIGN: usize = 8;
 #[cfg(all(any(target_arch = "x86",
               target_arch = "x86_64",
               target_arch = "aarch64",
-              target_arch = "powerpc64")))]
+              target_arch = "powerpc64",
+              target_arch = "mips64")))]
 const MIN_ALIGN: usize = 16;
```

Comment:
It should, [at least on all 64-bit GNU userlands](http://www.gnu.org/software/libc/manual/html_node/Aligned-Memory-Blocks.html).